src/m_menu.c

```diff
@@ -1678,12 +1684,13 @@ static menuitem_t SetupMenu[]=
 {
   // [FG] alternative text for missing menu graphics lumps
   {1,"M_GENERL",M_General,    'g', "GENERAL"},      // killough 10/98
-  {1,"M_COMPAT",M_Compat,     'p', "DOOM COMPATIBILITY"},
   {1,"M_KEYBND",M_KeyBindings,'k', "KEY BINDINGS"},
+  {1,"M_COMPAT",M_Compat,     'p', "DOOM COMPATIBILITY"},
   {1,"M_WEAP"  ,M_Weapons,    'w', "WEAPONS"},
   {1,"M_STAT"  ,M_StatusBar,  's', "STATUS BAR / HUD"},
   {1,"M_AUTO"  ,M_Automap,    'a', "AUTOMAP"},
   {1,"M_ENEM"  ,M_Enemy,      'e', "ENEMIES"},
+  {1,"M_SVOL"  ,M_Sound,      's', "SOUND VOLUME"},
```

Comment:
Bugs:
- Returning from "Sound Volume" should go to "Options", not the main menu.

Very minor personal preferences:
- Move "Screen Melt" to "General > Display Options" just above "Menu Background"
- Move "Screen Flashes" and "Level Brightness" to a new section called "Accessibility" between "Quality of Life" and "Miscellaneous"

Reply:
> * Returning from "Sound Volume" should go to "Options", not the main menu.

Fixed in https://github.com/fabiangreffrath/woof/pull/1402/commits/2adb0fa3262f9df938bc3220c513dedc53da5627

> * Move "Screen Melt" to "General > Display Options" just above "Menu Background"

"Screen Melt" is intended to be an accessibility option.

> * Move "Screen Flashes" and "Level Brightness" to a new section called "Accessibility" between "Quality of Life" and "Miscellaneous"

I think "Level Brightness" is a video feature. It was originally inspired by the Unity port.

Thanks for feedback!